components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/TestUtil.java

```diff
@@ -148,16 +144,20 @@ public static Application createSummaryApplication(Application app){
 
         return null;
     }
-    public String getIpAddressOfContainer(String aliasName) throws DockerCertificateException,
-            DockerException, InterruptedException {
-        DockerClient docker = DefaultDockerClient.fromEnv().build();
-        if (ipAddressMap.containsKey(aliasName)){
-            return ipAddressMap.get(aliasName);
-        }else{
-            String ipAddress = docker.inspectContainer(aliasName).networkSettings().ipAddress();
-            ipAddressMap.put(aliasName,ipAddress);
-            return ipAddress;
+
+    /**
+     * Utility for get Docker running host
+     * @return
+     * @throws URISyntaxException
+     */
+    public String getIpAddressOfContainer() throws URISyntaxException {
+        String ip = "localhost";
+        String dockerHost = System.getenv("DOCKER_HOST");
+        if (!StringUtils.isEmpty("DOCKER_HOST: " + dockerHost)) {
```

Comment:
we couldn't set this value as localhost for docker host contains tcp://localhost:2375 like tcp address